pyproject.toml

```diff
@@ -37,7 +37,6 @@ dnspython = "^2.4.2"
 [tool.poetry.group.dev.dependencies]
 pre-commit = "^3.3.3"
 tox = "^3.28.0"
-tox-poetry = "0.5.0" # deprecated, no sense in `^`
```

Comment:
Leave this in here, please. GitHub Actions still use it.

Reply:
Returned in 34d6a70c4441660287e56c07f4a5bd546759b7e9